classes/OJSSwordDeposit.inc.php

```diff
@@ -175,8 +175,12 @@ public function createPackage() {
 	 * @param $username string SWORD deposit username (i.e. email address for DSPACE)
 	 * @param $password string SWORD deposit password
 	 */
-	public function deposit($url, $username, $password) {
-		$client = new SWORDAPPClient();
+	public function deposit($url, $username, $password, $apikey = '') {
```

Comment:
(Generally we use `null` for default values, rather than an empty string. But that's being pretty pedantic.)

Reply:
Done.